src/common/user-store/preferences-helpers.ts

```diff
@@ -306,6 +306,36 @@ const updateChannel: PreferenceDescription<string> = {
   },
 };
 
+export enum ExtensionRegistryLocation {
+  DEFAULT = "default",
+  NPMRC = "npmrc",
+  CUSTOM = "custom",
+}
+export type ExtensionRegistry = {
+  location: ExtensionRegistryLocation.DEFAULT | ExtensionRegistryLocation.NPMRC;
+  customUrl?: undefined;
+} | {
+  location: ExtensionRegistryLocation.CUSTOM,
+  customUrl: string;
+};
+
+export const defaultExtensionRegistryUrl = "https://registry.npmjs.com";
```

Comment:
It says the [here]( https://docs.npmjs.com/cli/v8/using-npm/registry) that the default domain is .org

> npm is configured to use the npm public registry at https://registry.npmjs.org by default.

Should we use the same as default?

Reply:
Interesting, both work... maybe we should then